tools/devtools/wndt-bootstrap.mjs

```diff
@@ -0,0 +1,13 @@
+import {createWndtBanners, createWndtBlogPosts} from './utils.mjs';
+import {config} from 'dotenv';
+
+config();
+
+if (!process.env.DEVTOOLS_VERSION) {
+  throw 'Please make sure have a .env file with parameter DEVTOOLS_VERSION="xxx".';
+}
+
+// @ts-ignore
+createWndtBanners(process.env.DEVTOOLS_VERSION, ['en']);
```

Comment:
Technically there would be `const {defaultLocale} = require('site/_data/site.json');` but since you need to hardcode locales in other places I think we shouldn't bother.